src/components/timeline/TimelineDetailsExpander.js

```diff
@@ -0,0 +1,28 @@
+import { DownOutlined, UpOutlined } from '@ant-design/icons';
+import React, { useState } from 'react';
+
+const TimelineDetailsExpander = props => {
+	const { children } = props;
+	const [isVisible, setIsVisible] = useState(false);
+
+	return (
+		<>
```

Comment:
Is the whitespace here a github thing? Because I don't think that what I'm seeing is what you meant.